runtime/src/main/resources/jasperreports.properties

```diff
@@ -1,2 +0,0 @@
-net.sf.jasperreports.xpath.executer.factory=net.sf.jasperreports.xalan.util.XalanXPathExecuterFactory
```

Comment:
I think you can delete this file?

Reply:
Yep I will take care of it this tomorrow.